package.json

```diff
@@ -64,6 +64,7 @@
     "lib"
   ],
   "dependencies": {
+    "moment": "^2.14.1",
```

Comment:
You can actually request a formatted date from metaphysics for `date` fields: https://github.com/artsy/metaphysics/blob/master/schema/fields/date.js